README.md

```diff
@@ -1,16 +1,60 @@
+
+- Central DB as a service that cannot be fully configured for a particular use; data mesh is a pattern that is supplanting the one DB to rule them all of the past
+
+- Monolithic architectures that require the software to be released at best quarterly as opposed to minute by minute in a high performing organization
+
+Here are a few practical implications for the design of our software and services that we may not have fully embraced in the past:
+
+- *Automation is fundamental*: human interaction to tweak the settings means treating the containers as pets, not cattle.  That is an anti-pattern not consistent w/ cloud-native and Container Attached Storage.
+
+- *Design for failure*:  Kubernetes and the cloud-native pattern assumes that everything will fail at some point. Chaos engineering such as the Litmus Chaos project we started and developed into a viable project enforces the need to design for failure by creating failures in systems.
+
+- *GitOps*: Building off the importance of automation - the desired state of the environment is in the repository, i.e. GitOps is fundamental.
+
+- *Loosely Coupled* may be the two most important words in [the CNCF’s definition of cloud native](https://github.com/cncf/toc/blob/main/DEFINITION.md). The need for loose coupling is one reason why our users like Steven Bower do not want to use shared storage. One video we have in the welcome and onboarding slides is [this interview with Steven](https://www.youtube.com/watch?v=z_LbRfDKPvE).
+
+- *APIs and Declarative approach*: Loosely coupled and the existence of many sub-components suggests there are clear APIs including within the product and typically declarative approaches are used
+
+- *Cloud operations including auto-scaling*: Cloud-native also tends to imply cloud scalability - the ability to scale UP and scale DOWN.
+
+We will know we are succeeding in enabling our personas when a standard user of Kubernetes can grab OpenEBS and:
+
+- *grok it* -- what it is supposed to do, how it is designed, how it is explained
+
+- and *operate it* -- knowing that it will behave in a cloud-native or container attached storage way
+
+…while the Kubernetes SREs within the Platform Team can support the use of OpenEBS at scale because they will be able to do their job. We have more work to do here in defining what these platform teams require -- thankfully we are in frequent contact with them at places such as Bloomberg, Flipkart and Optoro. 
+
+Last but certainly not least -- we are working with data and increasingly important data that needs to be secured at least for some period of time.  We must avoid losing data, understanding that solutions such as flavors of OpenEBS LocalPV themselves by design do nothing to secure the data.  Remember the old saying, when our users lose their data they typically lose their jobs.
```

Comment:
@bobek - slight modification to the statement here. 

Last but certainly not least - we are working w/ data and increasingly important data that needs to be secured at least for some period of time.  We must avoid losing data, understanding that solutions such as flavors of OpenEBS LocalPV themselves by design do nothing to secure the data; _this should not be construed as an argument against LocalPV - far from it._  Remember the old saying, when our users lose their data they typically lose their jobs.

Reply:
This looks redundant to me. We are saying, that LocalPV is designed not to protect data by itself. Like the "by design" is an important statement as we are saying, we want it this way.

I read is as -- we need to leverage strengths and be aware of limitations of particular storage engines.